### PR TITLE
[DPEDE-337] Add state icon tooltip in expansion panel html + web component

### PR DESCRIPTION
**Pull request by lumdmtr**
https://ctl.atlassian.net/browse/DPEDE-337

### CODE REVIEW

File: src/chi/components/expansion-panel/expansion-panel.scss [hunk 1]
```diff
@@ -295,6 +295,18 @@ $transition-duration: 0.2s;
         padding-left: 2.5rem;
       }
     }
+    i.chi-icon {
```
Comment (jllr) [inline src/chi/components/expansion-panel/expansion-panel.scss:298]:
```suggestion
    .chi-icon.state {
```

File: src/chi/components/expansion-panel/expansion-panel.scss [hunk 2]
```diff
@@ -295,6 +295,18 @@ $transition-duration: 0.2s;
...
+      display: none;
+    }
+
+    &.-done {
+      i.chi-icon {
+        display: inline;
```
Comment (jllr) [inline src/chi/components/expansion-panel/expansion-panel.scss:304]:
```suggestion
      .chi-icon.state {
        display: block;
```

File: src/chi/components/expansion-panel/expansion-panel.scss [hunk 3]
```diff
@@ -368,8 +363,10 @@ $transition-duration: 0.2s;
 
       &.-done {
         & > .chi-epanel__header {
-          &::before {
-            left: 1.5rem;
+          i.chi-icon {
+            display: inline;
```
Comment (jllr) [inline src/chi/components/expansion-panel/expansion-panel.scss:367]:
```suggestion
          .chi-icon.state {
            display: block;
```

**Review by jllr — CHANGES_REQUESTED** [review]

File: src/custom-elements/src/components/expansion-panel/expansion-panel.tsx [hunk 4]
```diff
@@ -34,6 +38,11 @@ export class ExpansionPanel {
    */
   @Prop({ reflect: true }) stateIcon = false;
 
+  /**
+   * to render a state tooltip
```
Comment (josecarlosgonzalezv) [inline src/custom-elements/src/components/expansion-panel/expansion-panel.tsx:42]:
```suggestion
   * To render a state tooltip
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/custom-elements/src/components/expansion-panel/expansion-panel.tsx (hunk 4 above)
Comment (lumdmtr) [reply src/custom-elements/src/components/expansion-panel/expansion-panel.tsx:42]: @josecarlosgonzalezv  why should it be capitalized here if for other properties it's lower-cased

**Review by jllr — APPROVED** [review]

**Pull request merged**